Victor_Trush/02.Git.Local/README.md

```diff
@@ -0,0 +1,554 @@
+02.GIT.Local
```

Comment:
please use github markdown: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
but I used it and everything displayed correctly in my repository

Reply:
It is reST, not official  github markdown